readme.md

```diff
@@ -198,6 +198,15 @@ In addition to `redux` --> `mobx-state-tree`, we've also transitioned to using `
 
 In Bowser, TypeScript is fully set up, so if you know TS, all you need to do is start coding!
 
+## Upgrade 
```

Comment:
Can you move this above TypeScript? Right now it's inserted between two TypeScript sections.

Reply:
Ooops, fixed.